robot/basestation/static/js/app.js

```diff
@@ -260,3 +261,10 @@ plays an error sound once -> warning will not play if the document has not been
 function errorSound (){
   new Audio(ERROR_SOUND_PATH).play()
 }
+
+/*
+plays an error sound once -> warning will not play if the document has not been interacted with yet
```

Comment:
comment is confusing, it says its error sound but its called warning?

Reply:
Removed warningSound and redirected errorSound to new sound